test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
I was thinking that we can somehow run the existing topology suit in two modes. Was discussing how to do it best with @kostja. He had some ideas.

Reply:
Is that a blocker for the PR?

BTW. The test I added here would fail in a flaky way if we tried to run it in gossiper mode, with errors such as "decommission is still running" (when you try to perform the next decommission) etc.

Reply:
The ideas are compatible. E.g. have markers like `raft_topology_only` and `gossip_topology_only`. Unmarked tests run in both modes.

Reply:
They are. But we still managed to have an argument about it.

Reply:
I think they aren't, since my suggsetion is that there is no default. One has to specify modes explicitly for each test. This way we don't  bloat the coverage and only run relevant tests in both modes.